opendatahub-io/data-science-pipelines-operator
# add aggregate view role

## The issue resolved by this Pull Request:
Resolves #417 

## Description of your changes:
Adds a view role to be included in the view cluster role aggregation.

Additional minor updates to sort kustomization file for easier maintenance and adding optional kustomize kind/api reference.

## Testing instructions
After applying role to the cluster, the DSPA objection should be part of the clusterrole view:

```sh
oc get clusterrole view -o yaml | grep datasciencepipelines
```

## Checklist
